src/Spd.Manager.Printing/Documents/TransformationStrategies/PersonalLicencePreviewTransformStrategy.cs

```diff
@@ -0,0 +1,247 @@
+            }
+        }
+
+        return destImage;
+    }
+}
+
+public record PersonalLicencePreviewTransformRequest(Guid LicenceId) : DocumentTransformRequest;
+public record LicencePreviewJson()
+{
+    [JsonPropertyName("licenceNumber")]
+    public string LicenceNumber { get; set; } = null!;
+
+    [JsonPropertyName("licenceType")]
+    public string LicenceType { get; set; } = null!; //Armoured Vehicle Permit, Body Armour Permit, Security Worker Licence, Security Business Licence
+
+    [JsonPropertyName("photo")]
+    public string Photo { get; set; } = null!; //only apply to personal licence, no photo for biz licence
+
+    [JsonPropertyName("licenceCategories")]
+    public IEnumerable<string> LicenceCategories { get; set; } = Enumerable.Empty<string>();//["Security Guard", "Locksmith"], only has value for swl and biz licence
+
+    [JsonPropertyName("applicantName")]
+    public string ApplicantName { get; set; } = null!; //firstname middlename1 lastName
+
+    [JsonPropertyName("doingBusinessAsName")]
+    public string? DoingBusinessAsName { get; set; } //only apply to biz licence
+
+    [JsonPropertyName("issuedDate")]
+    public string? IssuedDate { get; set; } //yyyy-MM-dd
+
+    [JsonPropertyName("expiryDate")]
+    public string? ExpiryDate { get; set; } //yyyy-MM-dd
+
+    [JsonPropertyName("mailingAddress1")]
+    public string? MailingAddress1 { get; set; }
+
+    [JsonPropertyName("mailingAddress2")]
+    public string? MailingAddress2 { get; set; }
+
+    [JsonPropertyName("city")]
+    public string? City { get; set; }
+
+    [JsonPropertyName("provinceState")]
+    public string? ProvinceState { get; set; }
+
+    [JsonPropertyName("postalCode")]
+    public string? PostalCode { get; set; }
+
+    [JsonPropertyName("country")]
+    public string? Country { get; set; }
+
+    [JsonPropertyName("branchOffices")]
+    public IEnumerable<BranchAddress> BranchOffices { get; set; } = Enumerable.Empty<BranchAddress>(); //only apply to biz licence
+
+    [JsonPropertyName("conditions")]
+    public IEnumerable<string> Conditions { get; set; } = Enumerable.Empty<string>(); //only apply to personal licence, no conditions for biz licence
+
+    [JsonPropertyName("SPD_CARD")]
+    public SPD_CARD? SPD_CARD { get; set; } //only apply to personal licence, no spd_card for biz licence
+}
+
+public record SPD_CARD
+{
+    [JsonPropertyName("eyes")]
+    public string? Eyes { get; set; }
+
+    [JsonPropertyName("hair")]
+    public string? Hair { get; set; }
+
+    [JsonPropertyName("height")]
+    public string? Height { get; set; } //175cm
+
+    [JsonPropertyName("weight")]
+    public string? Weight { get; set; } //67kg
+
+    [JsonPropertyName("temporaryLicence")]
+    public bool TemporaryLicence { get; set; }
+
+    [JsonPropertyName("cardType")]
+    public string? CardType { get; set; } //SECURITY-WORKER-AND-ARMOUR
+}
+
+public record BranchAddress

```

Comment:
Please add line breaks on this fields as you did in the previous ones

Reply:
?

Reply:
ok, fixed